src/components/EdcHelp.tsx

```diff
@@ -55,11 +67,7 @@ export function EdcHelp(props: EdcHelpProps): JSX.Element {
           </Popover>
         }
       >
-        <i
-          className={
-            data.icon + ' help-icon' + (finalProps.dark ? ' on-dark' : '')
-          }
-        />
+        {getDisplayComponent(data.icon, finalProps)}
```

Comment:
Create a component should be better : edc-icon for example. This component will focus on the icon display according to the properties